bc_obps/reporting/api/report_operation.py

```diff
@@ -0,0 +1,52 @@
+from typing import Literal
+
+from django.http import HttpRequest
+from reporting.constants import EMISSIONS_REPORT_TAGS
+from reporting.schema.generic import Message
+from service.error_service.custom_codes_4xx import custom_codes_4xx
+
+from .router import router
+from reporting.api.permissions import (
+    approved_industry_user_report_version_composite_auth,
+    approved_authorized_roles_report_version_composite_auth,
+)
+from ..schema.report_operation import ReportOperationDataSchema, ReportOperationSchemaOut
+from ..service.report_operation_service import ReportOperationService
+
+
+def should_show(purpose: str, excluded_purposes: list) -> bool:
+    return purpose not in excluded_purposes
```

Comment:
This doesn't seem to be used anywhere, we should remove it or document it at least